### PR TITLE
Sidebar sublevels,color grid in docs and blog, tables in docs (responsive)

### PR DESCRIPTION
**Pull request by roo-shy**

### CODE REVIEW

**Comment by roo-shy** [comment]
@kvz Please review this PR and let's merge it if it's good. As it gets big. I'll start sticky header in a new PR and fix responsive tables and a few things about sidebar: 

screen shot 2016-12-20 at 11 28 48 pm



screen shot 2016-12-20 at 11 03 41 pm

screen shot 2016-12-20 at 11 04 26 pm

File: docs/decorators/clustering/consul_acl.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
 </table>
 
 <h2>Author</h2>
-Steve Gargan (@sgargan)
+<p>Steve Gargan (@sgargan)</p>
```
Comment (kvz) [inline docs/decorators/clustering/consul_acl.html]: This will just be overwritten, if this needs fixing I will have to deal with that in the importer.

**Comment by kvz** [comment]
Looking good @roo-shy, just one non-critical comment, I'll merge this in now.

One thing I noticed with the screenshare is there is (to me) some confusion about the different styling of items in the docs sidebar. I'd like to have one style for folders, and one style for "leaf-nodes" meaning items that cannot expand/collapse but will instead navigate you away to a new page. I think it will help the UX so that there are no surprises when you will navigate away from the page.

Currently I noticed there is one style for toplevel, and one style for every level beneath that?
I think I already add `leaf` classes or sth similar so that should make it easier to apply the idea I had re: this styling.

**Comment by roo-shy** [comment]
Ok sorry for misinterpreting I'll take care of it 👌

**Pull request merged**